### ISSUE TITLE
how to use keystone when there is already a User table in database?

### ISSUE THREAD

**Issue by ajayvishwanath** [keystonejs/keystone#1583]

**Comment by lojack** [comment]
I would recommend setting the [Schema's collection name](http://mongoosejs.com/docs/guide.html#collection)

**Comment by JedWatson** [comment]
Can you explain what you're trying to do in more detail please?

You can change the model name that Keystone uses for Users (set `user model`), or entirely replace the auth & session middleware with your own implementation if the structure of your User collection doesn't match what Keystone expects (in terms of `email`, `password`, etc)

**Comment by ajayvishwanath** [comment]
Hi Jed, thanks for the comment. This is what I am trying to achieve:

I have designed a set of NodeJs(Express+Mongoose) API's for a mobile app would contain a large user base. To keep track of these users, I have created a table known as "User".

Now, I want to implement a CMS to manage these users, the media etc. I found that keystone fulfilled most of my needs, except that it also has its own User table/schema. My question is, how do I differentiate my "user" from the keystone cms' "user" (super-admin)? Can keystone be used despite this ambiguity?
